GraalVM/CE/1.0.0-rc7/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 2015 Oracle and/or its affiliates. All rights reserved.
+#
+
+FROM oraclelinux:7-slim
+
+# Note: If you are behind a web proxy, set the build variables for the build:
+#       E.g.:  docker build --build-arg "https_proxy=..." --build-arg "http_proxy=..." --build-arg "no_proxy=..." ...
+
+ARG GRAAL_VERSION=1.0.0-rc7
+ENV LANG=en_US.UTF-8
+
+ENV GRAALVM_PKG=https://github.com/oracle/graal/releases/download/vm-$GRAAL_VERSION/graalvm-ce-$GRAAL_VERSION-linux-amd64.tar.gz \
+    JAVA_HOME=/opt/graalvm-ce-$GRAAL_VERSION/ \
+    PATH=/opt/graalvm-ce-$GRAAL_VERSION/bin:$PATH:/opt/rh/llvm-toolset-7/root/usr/bin \
```

Comment:
Please remove the `$JAVA_HOME/bin` from `$PATH` as this is taken care of by the `alternatives` block below.